modules/installation-aws-permissions.adoc

```diff
@@ -227,7 +226,7 @@ If you use an existing VPC, your account does not require these permissions for
 
 [NOTE]
 =====
-If you use an existing VPC, your account does not require these permissions to delete network resources.
+If you use an existing VPC, your account does not require these permissions to delete network resources. Instead, your account only requires the `tag:UntagResources` permission to delete network resources if you use an existing VPC.
```

Comment:
Seems you could omit the "if you use an existing VPC" and the end since the note starts with that.